Source/Core/LongLatGeocoderService.js

```diff
@@ -0,0 +1,54 @@
+/*global define*/
+define([
+    './Cartesian3',
+    './defaultValue',
+    '../ThirdParty/when'
+], function(
+    Cartesian3,
+    defaultValue,
+    when) {
+    'use strict';
+
+    /**
+     * Provides geocoding through Bing Maps.
+     * @alias LongLatGeocoderService
+     *
+     */
+    function LongLatGeocoderService(options) {
+        options = defaultValue(options, defaultValue.EMPTY_OBJECT);
+        this.autoComplete = false;
+    }
+
+    LongLatGeocoderService.prototype.cancel = function() {
```

Comment:
Add this function to the `GeocoderService` interface if it is required to be implemented